cmd/node/config/lightClientP2P.toml

```diff
@@ -0,0 +1,89 @@
+        [Node.Transports.TCP]
+            ListenAddress = "/ip4/0.0.0.0/tcp/%d" # TCP listen address
+            PreventPortReuse = false
+
+    [Node.ResourceLimiter]
+        Type = "default autoscale" #available options "default autoscale", "infinite", "default with manual scale".
+        ManualSystemMemoryInMB = 0 # not taken into account if the type is not "default with manual scale"
+        ManualMaximumFD = 0 # not taken into account if the type is not "default with manual scale"
+
+# P2P peer discovery section
+
+# The following sections correspond to the way new peers will be discovered
+# If all config types are disabled then the peer will run in single mode (will not try to find other peers)
+# If more than one peer discovery mechanism is enabled, the application will output an error and will not start
+
+[KadDhtPeerDiscovery]
+    # Enabled: true/false to enable/disable this discovery mechanism
+    Enabled = true
+
+    # Type represents the kad-dht glue code implementation.
+    # "legacy" will define the first implementation.
+    # "optimized" represents the new variant able to connect to multiple seeders at once. This implementation also has
+    # a built-in timer that will try to automatically reconnect to the seeders (in case the seeders recover after a
+    # premature shutdown)
+    Type = "optimized"
+
+    # RefreshIntervalInSec represents the time in seconds between querying for new peers
+    RefreshIntervalInSec = 10
+
+    # ProtocolIDs represents the protocols that this node will advertise to other peers
+    # To connect to other nodes, those nodes should have at least one common protocol string
+    ProtocolIDs = [
+        "mvx-light-client",
+    ]
+
+    # InitialPeerList represents the list of strings of some known nodes that will bootstrap this node
+    # The address will be in a self-describing addressing format.
+    # More can be found here: https://github.com/libp2p/specs/blob/master/3-requirements.md#34-transport-agnostic
+    # Example:
+    #    /ip6/fe80::8823:6dff:fee7:f172/tcp/4001/p2p/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
+    #    /ip4/162.246.145.218/udp/4001/utp/ipfs/QmYJyUMAcXEw1b5bFfbBbzYu5wyyjLMRHXGUkCXpag74Fu
+    #
+    # If the initial peers list is left empty, the node will not try to connect to other peers during initial bootstrap
+    # phase but will accept connections and will do the network discovery if another peer connects to it
+    InitialPeerList = ["/ip4/127.0.0.1/tcp/9999/p2p/16Uiu2HAkw5SNNtSvH1zJiQ6Gc3WoGNSxiyNueRKe6fuAuh57G3Bk"]
+
+    # kademlia's routing table bucket size
+    BucketSize = 100
+
+    # RoutingTableRefreshIntervalInSec defines how many seconds should pass between 2 kad routing table auto refresh calls
+    RoutingTableRefreshIntervalInSec = 300
+
+[Sharding]
+    # The targeted number of peer connections
+    TargetPeerCount = 41
```

Comment:
I think this can be done on a separate PR, where we try to extract the sharder outside of the networkMessenger component, so we can give it instead as an argument, then we can have the same sharder for both networks (mainnet & light client)

Reply:
alright. Then we leave this file as it is?